packages/dso-toolkit/src/components/document-component-list/document-component-list.scss

```diff
@@ -0,0 +1,12 @@
+@use "../../variables/units";
+@use "../list";
+
+ul.dso-document-component-list {
```

Comment:
Deze code is copy pasted uit `document-component.scss`, maar deze regel code was eerst `.dso-document-components`.

De voorgeschreven markup was eerst:

```html
<dso-responsive-element class="dso-document-components">
  <ul>
    <li>
      <dso-document-component>
```

en wordt nu:

```html
<dso-responsive-element class="dso-document-components">
  <ul class="dso-document-list">
    <li>
      <dso-document-component>
```